Tasks/SonarQubePostTest/SonarQubePostTestImpl.ps1

```diff
@@ -75,3 +75,12 @@ function UploadSummaryMdReport
 		 Write-Warning "Could not find the summary report file $summaryMdPath"
 	}
 }
+
```

Comment:
This piece of code also existed but IsPrBuild had a different implementation that always returned false.